frontend/src/pages/notebookController/screens/admin/useAdminUsers.ts

```diff
@@ -29,21 +29,24 @@ const useAdminUsers = (): [AdminViewUserData[], boolean, Error | undefined] => {
     const notebook = notebookRunningState?.notebook ?? null;
     const isNotebookRunning = notebookRunningState?.isRunning ?? false;
 
+    const serverStatusObject = {
```

Comment:
Type all objects -- TS will infer your keys as a type and thus disable TS for your object.